cirq/google/calibration/phased_fsim.py

```diff
@@ -434,15 +434,37 @@ class IncompatibleMomentError(Exception):
     """Error that occurs when a moment is not supported by a calibration routine."""
 
 
-def try_convert_sqrt_iswap_to_fsim(gate: Gate) -> Optional[FSimGate]:
+@dataclasses.dataclass(frozen=True)
+class FSimGateCalibration:
```

Comment:
I don't think this is a good name.  I would use AdjustedFSimGate or CalibratedFSimGate.  We should also consider making this implement Gate, since it is basically a gate anyway.

If we make it a gate, it should probably go in cirq.google.ops.

Reply:
This is definitely not a gate. This is association between a gate and hardware gate to characterize.

It reflects the state in two realms: the `engine_gate` is a gate which is supposed to be characterized. The pair (`engine_gate`, `phase_exponent`) is a gate that is supposed to be used during the compilation after merging with characterization obtained for engine_gate. I do think it belongs to this package rather than `cirq.ops`. Maybe `EngineFSimGate` would be a better name though?

Reply:
Why can't the phase_exponent just be `engine_gate.exponent`? (or `.theta` I guess)

Reply:
Because engine_gate wouldn't be serializable to FSIM/SQRT_ISWAP gateset? This is not theta, this is shift of -2 * pi * phase_exponent  in chi of PhasedFSimGate.  Maybe we should discuss it in person next week...

Reply:
Will we need one of these classes for each type of correction we want to implement?

Reply:
My hope is that this will be solved in a better way until this becomes necessary.